### PR TITLE
Include replicated segment size property for datasources endpoint

### LINKED ISSUE #7733: Provide segment size property that includes replicated segments
### Description

I'd like to a new property `replicatedSize` to the tier-specific result of `druid/coordinator/v1/datasources?simple` and `/druid/coordinator/v1/datasources/{dataSourceName}` which would provide the segment size that includes replication.
### Motivation
Presently the datasources endpoint provides the unreplicated segment size of each tier within a datasource. A replicated segment size property for each tier would be useful for services to monitor total tier usage periodically.

### PR DESCRIPTION
**Pull request by a2l007**
Fixes #7733.

The following coordinator endpoints: 
`/druid/coordinator/v1/datasources?simple`
`/druid/coordinator/v1/datasources/{dataSourceName}`
will now have an additional property `replicatedSize` which is the total segment size including replication. This property is added to tier-specific results as well as overall segment results.
Added tests.

### CODE REVIEW

**Comment by vogievetsky** [comment]
How would you compute the same thing in the systems tables?

**Comment by vogievetsky** [comment]
Figured it out: https://github.com/apache/incubator-druid/pull/8043

**Comment by a2l007** [comment]
@vogievetsky Thanks for adding it to the console as well

File: docs/content/operations/api-reference.md [hunk 1]
```diff
@@ -162,15 +162,15 @@ Returns a list of datasource names found in the cluster.
 
 * `/druid/coordinator/v1/datasources?simple`
 
-Returns a list of JSON objects containing the name and properties of datasources found in the cluster.  Properties include segment count, total segment byte size, minTime, and maxTime.
+Returns a list of JSON objects containing the name and properties of datasources found in the cluster.  Properties include segment count, total segment byte size, replicated total segment byte size, minTime and maxTime.
```
Comment (clintropolis) [inline docs/content/operations/api-reference.md]: nit: lost a comma, 
> minTime, and maxTime.

File: docs/content/operations/api-reference.md [hunk 2]
```diff
@@ -162,15 +162,15 @@ Returns a list of datasource names found in the cluster.
...
 
 * `/druid/coordinator/v1/datasources?full`
 
 Returns a list of datasource names found in the cluster with all metadata about those datasources.
 
 * `/druid/coordinator/v1/datasources/{dataSourceName}`
 
-Returns a JSON object containing the name and properties of a datasource. Properties include segment count, total segment byte size, minTime, and maxTime.
+Returns a JSON object containing the name and properties of a datasource. Properties include segment count, total segment byte size, replicated total segment byte size, minTime and maxTime.
```
Comment (clintropolis) [inline docs/content/operations/api-reference.md]: same nit about comma

File: docs/content/operations/api-reference.md (hunk 1 above)
Comment (a2l007) [reply docs/content/operations/api-reference.md]: Thanks for reviewing. It turns out both are grammatically correct: https://www.grammarly.com/blog/comma-before-and/
Regardless, I've added the comma back :)

File: docs/content/operations/api-reference.md (hunk 2 above)
Comment (a2l007) [reply docs/content/operations/api-reference.md]: Fixed

File: docs/content/operations/api-reference.md (hunk 1 above)
Comment (clintropolis) [reply docs/content/operations/api-reference.md]: Heh, indeed, that's why i called it a 'nit', but oxford comma is the one true way imo! 😅 Thanks for fixing :+1:

**Comment by clintropolis** [comment]
Teamcity failure is due to master broken at the time of last merge, since the change was only to docs, going to go ahead and merge.

**Pull request merged**